lib/oembed/utils.py

```diff
@@ -7,12 +7,17 @@
 from htk.lib.oembed.constants import *
 from htk.utils.request import get_current_request
 
-def get_oembed_html(url):
+def get_oembed_html(url, autoplay=False):
     """Gets the oEmbed HTML for a URL, if it is an oEmbed type
     """
     oembed_type = get_oembed_type(url)
+
     if oembed_type:
-        html = get_oembed_html_for_service(url, oembed_type)
+        if oembed_type == 'youtube':
+            autoplay = autoplay
+            html = youtube_oembed(url, autoplay)
```

Comment:
no, when you call the function, if the function takes named arguments, invoke the function with name arguments

Reply:
so `html = youtube_oembed(url, autoplay=autoplay)`

when doing assignment in Python, 1 space around the `=`

when funcall, no spaces around the `=`

Reply:
no, do it exactly as I said here: https://github.com/hacktoolkit/django-htk/pull/270#discussion_r202843295

Python knows that LHS = the named arg, and RHS is the variable in scope. no confusion